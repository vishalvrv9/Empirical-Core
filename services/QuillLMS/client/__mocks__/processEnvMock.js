export default {
  env: {
    CDN_URL: 'http://cdn.quill.org'
  }
}
