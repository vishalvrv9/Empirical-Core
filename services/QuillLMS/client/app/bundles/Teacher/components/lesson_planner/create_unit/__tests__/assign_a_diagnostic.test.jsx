import React from 'react';
import { shallow } from 'enzyme';

import AssignADiagnostic from '../assign_a_diagnostic';

import processEnvMock from '../../../../../../../__mocks__/processEnvMock.js';
window.process = processEnvMock;

describe('AssignADiagnostic component', () => {

  it('should render', () => {
    expect(shallow(<AssignADiagnostic />)).toMatchSnapshot();
  });

});
