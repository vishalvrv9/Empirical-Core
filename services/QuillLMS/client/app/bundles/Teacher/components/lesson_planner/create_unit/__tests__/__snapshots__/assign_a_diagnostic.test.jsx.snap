// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`AssignADiagnostic component should render 1`] = `
<div
  className="text-center"
  id="assign-a-diagnostic-page"
>
  <h1>
    Choose which type of diagnostic you'd like to use:
  </h1>
  <div
    className="minis"
  >
    <a
      href="undefined/diagnostic/413/stage/1"
    >
      <assignment_type_mini
        bodyText="Find your students’ writing abilities with a 22 question diagnostic."
        directions="use intermittently"
        img="http://cdn.quill.org/images/shared/diagnostic_icon.svg"
        link="/diagnostic/413/stage/1"
        quantity={1}
        timeDuration="~20 Min."
        title="Sentence Structure Diagnostic"
        unit="Diagnostic"
      />
    </a>
    <a
      href="undefined/diagnostic/ell/stage/2"
    >
      <assignment_type_mini
        bodyText="Find your students’ writing abilities with a 22 question diagnostic for language learners."
        directions="use continuously"
        img="http://cdn.quill.org/images/shared/ell_diagnostic_icon.svg"
        link="/diagnostic/ell/stage/2"
        quantity={1}
        timeDuration="~25 Min."
        title="ELL Diagnostic"
        unit="Diagnostic"
      />
    </a>
    <a>
      <LaunchingSoonMini />
    </a>
  </div>
</div>
`;
